apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/pom.xml

```diff
@@ -12,7 +12,7 @@
     <dependencies>
         <dependency>
             <groupId>com.expediagroup.apiary</groupId>
-            <artifactId>kafka-events-common</artifactId>
+            <artifactId>kafka-serializable-events</artifactId>
```

Comment:
I've never been a huge fan in having "serializable" in the class names and thus not here but maybe that's just me. Other options I can think of would be calling the module "kafka-apiary-events" or "kafka-apiary-hive-events". What do others think? If we do that we should rename the classes accordingly.

Reply:
`kafka-apiary-events` sounds good to me

Reply:
though is it really an Apiary event anymore? It's the entire hive event, so perhaps `kafka-hive-events` or even just `kafka-events` will do?

Reply:
Well, it's not actually the *entire* Hive event as we've omitted some fields that couldn't (easily) be serialized (and it's not the actual HiveEvent class objects either). So I was just thinking of giving them the Apiary "branding" as they are the Apiary version of the Hive Events that we just happen to be sending over Kafka although there probably isn't anything Kafka-specific in the events module. Having said that we're only going to use these to/from Kafka so calling them `kafka-hive-events` would also be fine for the module name. Thoughts on the class names if we remove "serializable" from them? That could be "apiary" instead?

Reply:
As discussed, let's rename to `apiary-hive-events` as there is nothing Kafka-related in them and we could re-use these objects in other messaging systems outside of Kafka.

Reply:
Done.